frontend/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "license": "AGPL-3.0",
   "scripts": {
-    "build": "cross-env NODE_ENV=production ./build.sh",
-    "build:mock": "cross-env NEXT_PUBLIC_USE_MOCK_API=true ./build.sh --keep-mock-api",
-    "build:test": "cross-env NODE_ENV=test NEXT_PUBLIC_TEST_MODE=true ./build.sh --keep-mock-api",
+    "build" : "cross-env NODE_ENV=production dotenv -- ./build.sh",
+    "build:mock" : "cross-env NEXT_PUBLIC_USE_MOCK_API=true dotenv -- ./build.sh",
+    "build:test" : "cross-env NODE_ENV=test NEXT_PUBLIC_TEST_MODE=true dotenv -c test -- ./build.sh",
     "analyze": "cross-env ANALYZE=true yarn build --profile",
```

Comment:
When already using `dotenv` why do you define additional env vars with `cross-env`? `dotenv` has the `-v` parameter for this.